.vscode/settings.json

```diff
@@ -1,4 +1,11 @@
 {
+  "black-formatter.args": [
+    "--line-length",
+    "9999"
+  ],
+  "flake8.args": [
+    "--max-line-length=9999"
+  ],
```

Comment:
Note: I included this because ruff is configured to ignore long lines [which is actually also something we can discuss], but VS Code isn't, so it kept flagging the long lines as problems. At least that's what was happening on my setup.